share/harvesters/org_swbiodiversity.py

```diff
@@ -0,0 +1,127 @@
+                data['title'] = title
+            start = record.div.div
+            description = self.process_text(start)
+            if description:
+                data['description'] = description
+            if start:
+                body = start.find_all_next(style='margin-top:5px;')
+                body = list(map(self.process_text, body))
+
+                for entry in body:
+
+                    if 'Contact:' in entry:
+                        contact_dict = {}
+                        contact = entry.replace('Contact:', '').strip()
+                        contact_email = contact[contact.find("(") + 1:contact.find(")")]
+                        contact_name = contact.split('(', 1)[0].strip()
+                        if ', Curator' in contact_name:
+                            contact_name = contact_name.replace(', Curator', '').strip()
+                        if contact and contact_email and re.match(r"[^@]+@[^@]+\.[^@]+", contact_email):
+                            contact_dict['email'] = contact_email
+                        if contact_name:
+                            contact_dict['name'] = contact_name
+                        if contact_dict:
+                            data['contact'] = contact_dict
+
+                    if 'Collection Type:' in entry:
+                        collection_type = entry.replace('Collection Type: ', '')
+                        data['collection-type'] = collection_type
+
+                    if 'Management:' in entry:
+                        management = entry.replace('Management: ', '')
+                        if 'Last Update:' in management:
+                            management_update = management.split('Last Update:', 1)
+                            management = management_update[0]
+                            last_update = management_update[1]
+                            if last_update:
+                                data['last-update'] = last_update.strip()
+                        data['management'] = management.strip()
+
+                    if 'Usage Rights:' in entry:
+                        usage_rights = entry.replace('Usage Rights: ', '')
+                        data['usage-rights'] = usage_rights
+
+                    if 'Access Rights' in entry or 'Rights Holder:' in entry:
+                        access_rights = entry.replace('Access Rights: ', '').replace('Rights Holder: ', '')
+                        data['access-rights'] = access_rights
+
+                collection_statistics = start.find_all_next('li')
+                collection_statistics = list(map(self.process_text, collection_statistics))
+                data['collection-statistics'] = self.process_collection_stat(collection_statistics)
+
+            count += 1
+            yield identifier, data
+
+    def process_text(self, text):
+        return re.sub('<[^>]*>', '', str(text)).strip()
+
+    def process_collection_stat(self, list):
+        stat = {}
+        for item in list:
```

Comment:
Avoid using `list` as a variable name, it shadows the built in type `list`
[Here's a list of builtins/names to avoid](https://docs.python.org/3/library/functions.html).